tests/security_test.json

```diff
@@ -0,0 +1,19 @@
+{
+	"commands": [
+		{
+			"name": "apt-get update",
```

Comment:
hmm ok. i'm wondering if it might make more sense to (in the future) support something like a "multi-command", for use cases where we need to make sure a set of commands are run together. won't do it now, but if this same use case pops up in multiple places i'll revisit.